tennis-scheduler-web-app/src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from "react"
+
+import {Login} from './components/Login';
+import { Registration  } from './components/Registration';
 
 function App() {
+
+  const [currentForm, setCurrentForm] = useState('login');
+
+  const toggleForm = (formName) => {
+    setCurrentForm(formName);
+  }
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-         Hello from tennis scheduler
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {  currentForm === "login" ?  <Login onFormSwitch={toggleForm} /> :  <Registration onFormSwitch={toggleForm} />}
```

Comment:
Use react router for different routes of the app, its better solution.

Reply:
These two components are on the same path, that is the reason why I didn't use router here.  I'll use it for other pages if that's ok?